modules/catalog/src/main/java/org/apache/ignite/internal/catalog/descriptors/CatalogTableDescriptorSerializers.java

```diff
@@ -191,7 +195,8 @@ static class TableDescriptorSerializerV2 implements CatalogObjectSerializer<Cata
         public CatalogTableDescriptor readFrom(CatalogObjectDataInput input) throws IOException {
             int id = input.readVarIntAsInt();
             String name = input.readUTF();
-            long updateToken = input.readVarInt();
+            long updateTimestampLong = input.readVarInt();
+            HybridTimestamp updateTimestamp = updateTimestampLong == 0 ? MIN_VALUE : hybridTimestamp(updateTimestampLong);
```

Comment:
Why is this necessary? How can it happen?

Reply:
It's necessary for correct processing of `0` value that may be found in the old storage to convert it to timestamp.

Reply:
Please state it explicitly in the comment, otherwise it's hard to understand what's going on